README.md

```diff
@@ -11,6 +11,14 @@ python -m venv .venv
 source .venv/bin/activate
 pip install -r requirements.txt
 ```
+For Windows10;
+```bash
+python -m venv .venv
+.venv\Scripts\activate
+pip freeze > requirements.txt
```

Comment:
"pip install -r requirements.txt" is good. I'm new in gitHub so that I couldn't change "pip freeze > requirements.txt" to "pip install -r requirements.txt"

Reply:
Yep, we need pip install as opposed to freeze.

Reply:
Please submit a new PR with the changes